src/layouts/partial-cards.html

```diff
@@ -305,7 +305,7 @@
             stateObj.attributes.friendly_name !== 'default_view') {
           return stateObj.attributes.friendly_name;
         }
-        return this.computeLocationName(this.hass);
+        return 'Home';
```

Comment:
Why would this be hardcoded to Home?

Reply:
It's the fallback for `default_view` if `icon` and `name` are not set.
Current fallback is to show locationName...
```
group:
  default_view:
    view: yes
    icon: mdi:home 
    name: Default
```